build/templates/_library_interpreter.py.mako

```diff
@@ -75,6 +79,19 @@ class LibraryInterpreter(object):
     def __init__(self, encoding):
         self._encoding = encoding
         self._library = _library_singleton.get()
+        % if 'SetRuntimeEnvironment' in functions:
+        try:
+            runtime_env_ctype = platform.python_implementation()
+            version_ctype = platform.python_version()
```

Comment:
This variable is not ctype just a Python string, no?
The name is misleading, consider just making the function call directly in the set_runtime_environment invocation.